self-core-impl/src/main/java/com/selfxdsd/core/contributors/ProjectContributors.java

```diff
@@ -154,12 +152,16 @@ public Contributors ofProject(
      *
      * In the future, we might take more factors into account.
      * @param task Task requiring an assignee.
-     * @return Contributor or null if nobody is found.
+     * @return Contributor or null if nobody is found or Task's Project
+     * not matching ProjectContributors Project.
      * @checkstyle ReturnCount (30 lines)
      * @checkstyle Indentation (30 lines)
      */
     @Override
     public Contributor elect(final Task task) {
+        if (!this.project.equals(task.project())) {
+            return null;
```

Comment:
@criske I think we should thrown an ISE in this case, since it's not the correct flow -- the given Task should always correspond to the Project.